ChurchChallenge/ChurchChallenge/Model/Individual.swift

```diff
@@ -30,116 +30,12 @@ class Individual: Object, Decodable {
-                                IndividualsListTableViewController.numberOfImagesLoaded += 1
-                                print(url)
-                                if IndividualsListTableViewController.numberOfImagesLoaded == list.count {
-                                    UIApplication.shared.isNetworkActivityIndicatorVisible = false
-                                    completion(list)
-                                }
-                            }
-                        }
-                    }
-                }
-            } else {
-                UIApplication.shared.isNetworkActivityIndicatorVisible = false
-                completion(response.result.value)
-            }
-        }
-    }
     
-    /// Download single image from the web.
-    ///
-    /// - Parameters:
-    ///   - url: Location of image.
-    ///   - returnImage: UIImage returned back.
-    func dowmloadImage(url: String, returnImage: @escaping (UIImage) -> Void) {
-        Alamofire.request(url).responseImage { response in
-            if let error = response.error?.localizedDescription {
-                showAlert(title: "Load Failed", message: error)
-            }
-            if let image = response.result.value {
-                returnImage(image)
-            }
-        }
-    }
     
-    func getAffiliationImage() -> UIImage? {
-        switch self.affiliation {
-        case Affiliation.JEDI.rawValue:
-            return UIImage(named: Constants.JediOrder)
-        case Affiliation.RESISTANCE.rawValue:
-            return UIImage(named: Constants.RebelAlliance)
-        case Affiliation.SITH.rawValue:
-            return UIImage(named: Constants.Sith)
-        case Affiliation.FIRST_ORDER.rawValue:
-            return UIImage(named: Constants.FirstOrder)
-        default:
-            return nil
-        }
-    }
     
-    // Save to disk with Realm.
-    func save() {
-        do {
-            let realm = try Realm()
-            try realm.write {
-                realm.add(self)
-            }
-        } catch {
-            showAlert(title: "Save Failed", message: error.localizedDescription)
-        }
-    }
-    
-    // Load from disk with Realm.
-    func load() -> Results<Individual>? {
-        do {
-            let realm = try Realm()
-            return realm.objects(Individual.self).sorted(byKeyPath: "id")
-        } catch {
-            showAlert(title: "Load Failed", message: error.localizedDescription)
-            return nil
-        }
-    }
-    
-    // Delete all from disk with Realm.
-    func deleteAll() {
-        do {
-            let realm = try Realm()
-            try realm.write {
-                realm.delete(realm.objects(Individual.self))
-            }
-        } catch {
-            showAlert(title: "Delete All Failed", message: error.localizedDescription)
-        }
-    }
     
 }
 
 
-enum Affiliation: String, Decodable {
-    case JEDI, RESISTANCE, SITH, FIRST_ORDER
-}
 
 
```

Comment:
There is a lot of white space at the end of this class.

Reply:
Cleaned up white space.